src/attachment/test/files.test.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, fixture, elementUpdated } from "@open-wc/testing";
+import { Attachment } from "../index";
+
+const files = [
+  { name: "file.txt", size: "150" },
+  { name: "icon.jpg", size: "200" },
+];
+const invalidFileSizeFiles = [
+  { name: "file.txt", size: "text" },
+  { name: "icon.jpg", size: "200" },
+];
+describe("Attachment", () => {
+  describe("value", () => {
+    it("should be none checked items when not assinged on constructor", async () => {
```

Comment:
I think "should be none checked items when not assinged on constructor" not correct.
Please change it